get-started-with-go/includes/4-install-vs-code.md

```diff
@@ -1,3 +1,73 @@
 # Exercise - Set up your development environment
+You can use any text editor to write Go applications, but when you use an IDE like Visual Studio Code (VS Code), you write code faster. Without an IDE, you’d have to write every line from scratch, which at the beginning could help to get familiar with the syntax, but in the long-term, you might need more time to write an application. Therefore, we recommend using an IDE. For instance, you could start using VS Code, which is free and open-source, which has an extension for Go to help you write code faster.
+
+When you have VS Code and the Go extension, you get the following benefits:
+
+* Format applications following Go standards
+* Add package references as soon as you add a new line and save your code.
+* Remove package references that your application is not using anymore.
+* Debug your applications by allowing you to set breakpoints and use watchers at runtime
+* Provides suggested completions for code and package references
+* Hover information to get more details about the code referencing its documentation
+* Provides signature help like listing the parameters a function has
+* Navigate the project’s code easily when you have multiple files
+* Support for testing like generating test skeletons for functions
+
+Below are the instructions to install Visual Studio Code on your workstation.
+
+## Install Visual Studio Code on macOS
+
+**Step 1: Download Visual Studio Code**
+Head over to the Visual Studio Code page and click on the “Mac” box to download the app. Wait a few seconds. You’ll see a prompt to save the file locally.
+
+**Step 2: Move the app to the Applications folder**
+Locate the file you’ve just downloaded and open Finder. Drag and drop the Visual Studio Code file to the Applications folder in Finder.
+
+**Step 3: Start the app from the Applications folder**
+To start the Visual Studio Code app, double-click on the icon from the Applications folder.
+
+**Important!**
+You might see a warning that Visual Studio Code can't be opened because Apple can't check it for malicious software. If that happens, select OK to dismiss the message. Then right-click Visual Studio Code in the Applications folder and choose the Open menu. If you choose this option, Visual Studio Code should open without any further issues.
```

Comment:
```suggestion
> You might see a warning that Visual Studio Code can't be opened because Apple can't check it for malicious software. If that happens, select OK to dismiss the message. Then right-click Visual Studio Code in the Applications folder and choose the Open menu. If you choose this option, Visual Studio Code should open without any further issues.
```